site/src/atomics/spacing.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Spacing Atomics
+description: Spacing related atomic css classes for the Atlas Design System
+template: standard
+---
+
+# Spacing Atomics
+
+Spacing Atomics can be used to add margin or padding on elements. They are especially helpful when composing layouts.
+
+## Class Pattern
+
+`<css-property-name>-<value>-<screensize>`
+
+For general information about the pattern, visit the [Atomics page](https://github.com/microsoft/atlas-design/blob/main/css/src/atomics/README.md).
+
+The following `<css-property-names>-<values>-<screensizes>` are supported.
+
+Accepted property names
+
+- `margin`, `padding`
+- `margin|padding-inline`
+- `margin|padding-block`
+- `margin|padding-top|right|bottom|left`
+
+Accepted values
+
+- `none, xs, s, m, l, xl, xxl, xxxl`
+
+Accepted screensizes
+
+- `tablet, desktop, widescreen`
+
+## Usages
+
+Adding uniform spacing
```

Comment:
Let's make a heading -> `### Adding uniform spacing`

Reply:
Originally, I had headings for each section, but Will felt like there were too many headings, hence the current form.

Reply:
Hmm what about making them bold at least? Cause I really feel we need to highlight it somehow

Reply:
Made them bold.

Reply:
Looks better, but I still think semantically it should be heading 🤔

@wibjorn @asrivastava0308 thoughts?

Reply:
I was a bit concerned about the lacking of spacing around the "heading", but the bold seems to help.